ch_6_GPU_Acceleration/6_2_GPU_Baichuan2-7B.md

```diff
@@ -0,0 +1,110 @@
+# 6.2 Run Baichuan 2 (7B) on Intel GPUs
+
+You can use IPEX-LLM to load any ModelScope model for acceleration on Intel GPUs. With IPEX-LLM, PyTorch models (in FP16/BF16/FP32) hosted on ModelScope can be loaded and optimized automatically on Intel GPUs with low-bit quantization (supported precisions include INT4/NF4/INT5/FP8/INT8).
+
+In this tutorial, you will learn how to run LLMs on Intel GPUs with IPEX-LLM optimizations, and based on that build a stream chatbot. A popular open-source LLM [baichuan-inc/Baichuan2-7B-Chat](https://www.modelscope.cn/models/baichuan-inc/Baichuan2-7B-Chat) is used as an example.
+
+## 6.2.1 Load Model in Low Precision
+
+One common use case is to load a ModelScope model in low precision, i.e. conduct **implicit** quantization while loading.
+
+For Baichuan 2 (7B), you could simply import `ipex_llm.transformers.AutoModelForCausalLM` instead of `transformers.AutoModelForCausalLM`, and specify `load_in_4bit=True` or `load_in_low_bit` parameter accordingly in the `from_pretrained` function.
+
+For Intel GPUs, **once you have the model in low precision, set it to `to('xpu')`.**
+
+**For INT4 Optimizations (with `load_in_4bit=True`):**
+
+```python
+from ipex_llm.transformers import AutoModelForCausalLM
+
+# When running LLMs on Intel iGPUs for Windows users, we recommend setting `cpu_embedding=True` in the from_pretrained function.
+# This will allow the memory-intensive embedding layer to utilize the CPU instead of iGPU.
+# It is important to set `model_hub='modelscope'`, otherwise model hub is default to be huggingface
+model_in_4bit = AutoModelForCausalLM.from_pretrained(pretrained_model_name_or_path="baichuan-inc/Baichuan2-7B-Chat",
+                                                     load_in_4bit=True,
+                                                     trust_remote_code=True,
+                                                     use_cache=True,
+                                                     model_hub='modelscope')
+model_in_4bit_gpu = model_in_4bit.to('xpu')
+```
+
+> **Note**
+> IPEX-LLM has supported `AutoModel`, `AutoModelForCausalLM`, `AutoModelForSpeechSeq2Seq` and `AutoModelForSeq2SeqLM`.
```

Comment:
I think we supported more https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/src/ipex_llm/transformers/__init__.py#L19

Maybe list them all, or list the most important onces and add `, etc.` to the and